config/mailer.js

```diff
@@ -1,10 +1,6 @@
 module.exports = {
   transport: {
-    host: 'smtp.ethereal.email',
-    port: 587,
-    auth: {
-      user: 'g3jjujasexmtz4dl@ethereal.email',
-      pass: 'QF2bP2FcHa5cVCVPPT',
-    },
+    host: 'localhost',
```

Comment:
Do we want to configure this differently on a per-environment basis? Or are these settings ok for all? @giorgiosironi - what would you recommend?